priv/templates/nimble_template/assets/css_bootstrap/vendor/_bootstrap.scss

```diff
@@ -0,0 +1,46 @@
+// Configuration
```

Comment:
![image](https://user-images.githubusercontent.com/1896814/156495230-bc5a5740-b3bf-44b7-be1b-914fb036478a.png)
I read the PR description, but I don't get the idea of why to name the directories in an inconsistency way:
`css_boostrap` vs `nimble_css/nimble_js`
could you clarify ? 🙏

Reply:
css_bootstrap comes from @topnimble, and the `nimble_css and nimble_js` is just the addon.

If it's not clear, we can change to `bootstrap_css` :)

Reply:
Changed to bootstrap_css in https://github.com/nimblehq/elixir-templates/pull/161/commits/5d289a05b1228ed3be5833de63b92e7adfdb773e